index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Demo</title>
+</head>
+
+<body>
+  <div id="root"></div>
+  <script src="https://react-devtools-inline-initialize.netlify.app/react-devtools-inline-initialize.umd.js"></script>
```

Comment:
Ideally we move this `<script` also to the vite plugin (I am not sure if we will not break some React Devtools workflow).

Let's try it at the end.

Reply:
Tried moving it to the vite plugin, React Devtools warning popped up